tests/sceneManager_test.cpp

```diff
@@ -39,22 +40,73 @@ TEST_F(SceneManagerTest, EnergyCollectorPositionsCheck)
     ASSERT_EQ(manager.getEnergyCollectors().size(), constants::kPopulation) << "wrong numebr of energy collectors";
 
     RayHitData hitData;
-    Ray straightUp(core::Vec3(0, 0, 0), core::Vec3(0, 0, 1));
+    Ray straightUp(kVZero, core::Vec3(0, 0, 1));
 
     ASSERT_TRUE(performHitAtEnergyCollectors(straightUp, &hitData)) << "no hit: " << straightUp;
 
     core::Vec3 referenceCollisionPoint(0, 0, manager.simulatiorRadius() / 2 - manager.collectorRadius());
     ASSERT_EQ(hitData.collisionPoint, referenceCollisionPoint) << "point from: " << straightUp;
 
-    Ray parallelXAxis(core::Vec3(0, 0, 0), core::Vec3(-1, 0, 0));
+    Ray parallelXAxis(kVZero, core::Vec3(-1, 0, 0));
     ASSERT_TRUE(performHitAtEnergyCollectors(parallelXAxis, &hitData)) << "no hit: " << parallelXAxis;
 
-    Vec3 referenceCollisionPoint2(-(manager.simulatiorRadius() / 2 - manager.collectorRadius()), 0, 0);
-    ASSERT_EQ(hitData.collisionPoint, referenceCollisionPoint2) << " point from: " << parallelXAxis;
+    Vec3 referenceCollisionPoint2(
+        -(manager.simulatiorRadius() / 2 - manager.collectorRadius()), 0, 0);
+    ASSERT_EQ(hitData.collisionPoint, referenceCollisionPoint2)
+        << " point from: " << parallelXAxis;
 
-    Ray parallelYaxis(core::Vec3(0, 0, 0), core::Vec3(0, 1, 0));
-    ASSERT_TRUE(performHitAtEnergyCollectors(parallelYaxis, &hitData)) << "no hit: " << parallelYaxis;
+    Ray parallelYaxis(kVZero, core::Vec3(0, 1, 0));
+    ASSERT_TRUE(performHitAtEnergyCollectors(parallelYaxis, &hitData))
+        << "no hit: " << parallelYaxis;
 
     Vec3 referenceCollisionPoint3(0, manager.simulatiorRadius() / 2 - manager.collectorRadius(), 0);
-    ASSERT_EQ(hitData.collisionPoint, referenceCollisionPoint3) << " point from: " << parallelYaxis;
+    ASSERT_EQ(hitData.collisionPoint, referenceCollisionPoint3)
+        << " point from: " << parallelYaxis;
+
+    Ray atSixtyDegreesAlongX(kVZero, core::Vec3(std::cos(constants::kPi / 3), 0,
+                                                std::sin(constants::kPi / 3)));
+    ASSERT_TRUE(performHitAtEnergyCollectors(atSixtyDegreesAlongX, &hitData))
+        << "no hit: " << atSixtyDegreesAlongX;
+
+    Vec3 referenceCollisionAtSixty(
+        manager.simulatiorRadius() / 2 * std::cos(constants::kPi / 3) -
+            manager.collectorRadius() * std::cos(constants::kPi / 3),
+        0,
+        manager.simulatiorRadius() / 2 * std::sin(constants::kPi / 3) -
+            manager.collectorRadius() * std::sin(constants::kPi / 3));
+
+    ASSERT_EQ(hitData.collisionPoint, referenceCollisionAtSixty)
+        << "Invalid hit from " << atSixtyDegreesAlongX
+        << " to: " << referenceCollisionAtSixty;
+
+    Ray atThirtyDegreesAlongX(kVZero,
+                              core::Vec3(std::cos(constants::kPi / 6), 0,
+                                         std::sin(constants::kPi / 6)));
+    ASSERT_TRUE(performHitAtEnergyCollectors(atThirtyDegreesAlongX, &hitData))
```

Comment:
As I told you before: remove, this is no different than the case with 60 degrees.

Reply:
done